sql/V1_12__votes.sql

```diff
@@ -0,0 +1,39 @@
+ALTER TABLE IF EXISTS proposals
+DROP CONSTRAINT IF EXISTS proposals_proposal_id_ux;
+
+ALTER TABLE IF EXISTS proposals
+ADD CONSTRAINT proposals_proposal_id_ux UNIQUE (chain_num, proposal_id);
+
+CREATE TABLE IF NOT EXISTS
+  votes (
+    TYPE TEXT NOT NULL,
+    block_height BIGINT NOT NULL,
+    tx_idx SMALLINT NOT NULL,
+    msg_idx SMALLINT NOT NULL,
+    chain_num SMALLINT NOT NULL,
+    TIMESTAMP timestamptz,
+    tx_hash TEXT NOT NULL,
+    proposal_id BIGINT NOT NULL,
+    voter TEXT NOT NULL,
+    OPTION TEXT NOT NULL,
+    metadata TEXT NOT NULL,
+    submit_time timestamptz NOT NULL,
+    PRIMARY KEY (
+      chain_num,
+      block_height,
+      tx_idx,
+      msg_idx
+    ),
```

Comment:
So this primary key is not going to work when there are two votes. I can share a link here in a second but two votes for the same proposal hits a duplicate key violation error.

Reply:
I think we just need to add the voter.

Reply:
Thanks, I'll take a better look next week, oversights on my part with multiple voters.

Reply:
Here's the link: https://github.com/regen-network/indexer/actions/runs/6043549097/job/16400735995#step:5:639

Reply:
The db logs get pretty nasty when this occurs so may have trouble loading the db logs.

![Screenshot from 2023-08-31 17-01-45](https://github.com/regen-network/indexer/assets/12519942/cbf13509-180d-4258-a013-1a5148eb48fd)